### ISSUE TITLE
Accessibility Vocabulary 

### ISSUE THREAD

**Issue by MRuecklCC** [openeduhub/oeh-metadata-vocabs#18]
Currently, the [accessibility vocabulary](https://github.com/openeduhub/oeh-metadata-vocabs/blob/master/accessibilitySummary.ttl) holds 5 different choices: 

 - A
 - AA
 - AAA
 - WCAG
 - BITV

Unfortunately, I was not able to find any documentation about when exactly use which choice. Reading about [BITV](https://www.bitvtest.de/bitv_test.html) and WCAG, I found that BITV seems to be a superset of WCAG, i.e. if a site is BITV compliant it would also have a AAA WCAG rating. This makes the current definition of the value space slightly confusing:

- Why is there a WCAG value?
- There is no value for "not even A", i.e. a site that does not fulfill even the lowest standards,
- We could change to (in ascending order):
  - B (or some other name, no compliance)
  - A (WCAG)
  - AA (WCAG)
  - AAA (WCAG)
  - BITV

IMHO, it is very important to distinguish the scenario, where a rating is not available (missing meta-datum) and where a site is not compliant with any rating (currently also a missing meta-datum), so we should definitively add a "B" rating value.
